client/src/ui/styles/TwoColumnLayout.scss

```diff
@@ -0,0 +1,18 @@
+.two-column-layout {
```

Comment:
this + `site-content__centered` might mean we can also close #36 now

Reply:
added the extra Closes flag